handlers/controller.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Simple ping procedure to ensure server is listening and can talk.
+ * @param req request.
+ * @param res response.
+ * @returns {Promise<void>} n/a
+ */
+
+const { models, connectDb } = require ('../dal/database');
+
+let ping = async (req, res) => {
+    res.status(200).send('Ping!');
+};
+
+let getProduct = async (req, res) => {
+    let id = req.params.id;
+
+    connectDb().then(async () => {
+        // TODO get product logic goes here
+        // ***** E.G. ******
+        // models.Product.findById(id).
+        // then((doc) => {
+        //         if(!doc) return res.status(404).send();
+        //         else res.status(200).send({ doc });
+        //     })
+        //     .then(() => {return res.status(200).send('blah')})
+        //     .catch(() => res.status(500).send());
+        // *****************
+        res.status(200).send({ payload: null });
```

Comment:
Are we keeping the null payload line as a placeholder for now?

Reply:
Correct.